content/grants/proposals/feed-aggregator.md

```diff
@@ -0,0 +1,96 @@
++++
+title = "Urbit Feed Aggregator"
+date = 2021-07-16
+[taxonomies]
+grant_type = ["proposals"]
+grant_category = ["App Dev: Other"]
+[extra]
+image = ""
+description = "A feed aggregation application"
+reward = 5
+assignee = "~mirtyl-wacdec"
+grant_id = "P0043"
+completed = false
+link = ""
++++
+
+### Project Name
+
+Urbit Feed Aggregator
+
+## Purpose
+
+Urbit as it stands now (June 2021) is a very functional social network, but it's not yet making use of all the strengths of its computing model.
+As a personal server Urbit is the perfect platform to build a centralized social media consumption application.
```

Comment:
Assuming they mean "decentralized" here?

```suggestion
As a personal server Urbit is the perfect platform to build a decentralized social media consumption application.
```

Reply:
I think centralized is correct. The data from the disparate feeds is centralized on your Urbit server.